README.md

```diff
@@ -62,13 +62,12 @@ _Note: `4.5.1` can be replaced by any [released](https://github.com/pixijs/pixi.
 
 ### Demos ###
 
-- [WebGL Filters!](http://pixijs.github.io/pixi-filters/examples/)
+- [Filters Demo](http://pixijs.io/pixi-filters/tools/demo/)
 - [Run pixie run](http://www.goodboydigital.com/runpixierun)
```

Comment:
Is this the link for Run Pixie Run? 
https://runpixierun.tresensa.com/

And some nitpicking: let's change "Run pixie run" to "Run Pixie Run" for some consistency.